### ISSUE TITLE
ENH-000124608: Displayed Symbol ID Code can not be copied and Search displays a invalid code that is less than 15 characters.

### ISSUE THREAD

**Issue by cdufault** [Esri/military-symbology#386]
When creating a new symbol, for a new data item, it would be very helpful to be able to copy the SIDC value generated. This could then be used in a the field in the a feature class that is being edited. When generating a symbol the SIDC values displayed do show the full 15 characters needed to use the renderer in pro. 
The work flow:

1. Enter in the search what type of symbol is needed. 
2. Then configure the symbol to look as needed. 
3. The codes displayed in the attached images on the search tab do not have the full symbol id 15 characters.
4. There is also no where to copy the SIDC code, which is displayed correctly in the symbol tab, from the tool to add to my feature in the data.

Please make all the places the tool displays the codes to be the proper 15 characters. Also please make a place or button to copy the generated 15 character SIDC value.
![image](https://user-images.githubusercontent.com/4854714/62793683-fe84a280-ba86-11e9-9267-5b635ca5fc09.png)

![image](https://user-images.githubusercontent.com/4854714/62793201-d6e10a80-ba85-11e9-9aa9-438cfe8dcaec.png)

**Comment by csmoore** [comment]
Added copy to attribute table and SymbolIdCode attribute to the table in PR #390 

Should look/work like this now:
![image](https://user-images.githubusercontent.com/3090809/63182247-20020300-c020-11e9-9cfe-39db3a531c17.png)

**Comment by cdufault** [comment]
Got the enhancement number for you. ENH-000124608 in sales force now :)

**Comment by dfoll** [comment]
@csmoore I have tested this. Only one thing I would like to ask for  before merging. I noticed that if you set the Country Code on the Label tab, it does not update the SIDC. For Bc2 this is positions 13/14, I did not do the research to see what the positions are for the other standards. Let me know if you want me to fetch that information for you.

**Comment by dfoll** [comment]
- `Copy` from the MSE attribute table copies the entire row (as designed)
- SIDC is not stored as an attribute in the gdb (known issue #317, waiting to implement this until we design/understand a workflow that does not have a risk of a feature's SIDC and that attributes getting out of sync)

**Comment by dfoll** [comment]
### Doc QC:
- [ ]  [Defense Whats new](https://solutionsdev.arcgis.com/defense/what-is-new/) -- after [PR 8017](https://github.com/ArcGIS/arcgis-solutions-website/pull/8017) is reviewed and merged
- [ ] [Military Symbol Editor October Resources](https://solutionsdev.arcgis.com/defense/help/military-symbol-editor/resources/october-2019-military-symbol-editor.htm)
- [ ] \DefenseSolutions\MilitarySymbolEditor\October2019\Dev\147\
- [ ] [Military Tools October Resources](https://solutionsdev.arcgis.com/defense/help/military-tools/resources/october-2019-military-tools-for-arcgis.htm)
- [ ] \DefenseSolutions\MilitaryToolsforArcGIS\October2019\Dev\341\

**Comment by csmoore** [comment]
@dfoll PR #390 updated in commit https://github.com/Esri/military-symbology/pull/390/commits/3e2e212e08fa4ae65004b555e871a89a25195aa1 with fix for country code in SIDC

**Comment by dfoll** [comment]
@csmoore I have tested the updated #390. I saw issues with the update.

- for the standards that use alpha for the SIDC, when you set the country code, the SIDC updates correctly, however, it holds onto that country code as part of the SIDC when you create a new symbol (even though the country code attribute is set to null). This happens even between projects. 

- for the standards that use numeric for the SIDC the SIDC is still not being updated when you set a country code
